backend/src/app.service.ts

```diff
@@ -2,7 +2,7 @@ import { Injectable } from '@nestjs/common';
 
 @Injectable()
 export class AppService {
-  getHello(): string {
-    return 'Hello World!';
+  getHealth(): string {
```

Comment:
What happens when the app is not running? Just a 4xx error?

Reply:
I think it depends on where we deploy, but I think EC2 should be 404 (?)